kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF/mii-initial-d1-JRF-v1.yaml

```diff
@@ -93,7 +93,7 @@ spec:
       - name: USER_MEM_ARGS
```

Comment:
These files yaml files should be generated from a template and then checked in after generating them.  It's clear they were not generated from a template, otherwise all of the model YAMLs would have changes instead of just a couple of them. If the template and the sample yaml don't match, then the WKO integration tests will detect the discrepancy and fail.  To generate new versions of the file from a template see the 'SAMPLE REGENERATION' step in `operator/integration-tests/model-in-image/mii-sample-wrapper/README`

Reply:
Will do. Thanks.